val x = lazy exception A

x < 2

match x 
  case _ => 1
end

